django_freeradius/api/views.py

```diff
@@ -168,10 +171,11 @@ def post(self, request, *args, **kwargs):
 
     def create(self, request, *args, **kwargs):
         is_start = request.data['status_type'] == 'Start'
+        data = request.data.copy()  # import because request objects is immutable
         for field in ['session_time', 'input_octets', 'output_octets']:
             if is_start and request.data[field] == '':
-                request.data[field] = 0
-        serializer = self.get_serializer(data=request.data)
+                data[field] = 0
```

Comment:
why not adding the groupname here directly instead of overriding `perform_create`? Wouldn't it be simpler?

Reply:
I did the perform_create part, because I've read several places that it's better to do so. 
And I had more freedom as to adding try-except. If you want I can get rid of this.